vhost-device-scmi/src/devices/common.rs

```diff
@@ -230,6 +232,15 @@ pub struct Sensor {
     /// Sensors can be enabled and disabled using SCMI.  [Sensor]s created
     /// using [Sensor::new] are disabled initially.
     enabled: bool,
+
+    /// Sensor notification can be enabled or disabled when frontend send SENSOR_CONTINUOUS_UPDATE_NOTIFY.
+    notify_enabled: bool,
+    /// If this sensor support notifing fronted initiatively, it should record
+    /// File there.
+    pub notify_dev: Option<File>,
+
+    /// Store this sensor's id in Strcut Sensor
```

Comment:
In file "vhost-device-scmi/src/devices/scmi.rs", we refactor function "insert" to return a identify.
The structure "DeviceIdentify" contains "protocol id, device id", it is the key in hashmap for us to find out which device is notifying. (By protocol id, and device id, it can locate specified device).

Reply:
I see. How about to change the doc string to let's say:
``/// Sensor id, to identify the sensor in notification lookup.``

Reply:
I will change it.